README.md

```diff
@@ -2,12 +2,15 @@
 
 This is an open source emulator for [Azure Event Grid](https://learn.microsoft.com/en-us/azure/event-grid/overview) that supports only the [push delivery](https://learn.microsoft.com/en-us/azure/event-grid/push-delivery-overview) model. Based on ASP.NET Core, this emulator provides a cross-platform experience for developers wanting to try Azure Event Grid easily in a local environment without having to deploy to Azure.
 
+The emulator supports two Events delivery formats: Push model and Pull model.
+
 This project is not affiliated, associated, authorized, endorsed by, or in any way officially connected with Microsoft.
 
 ## Features
 
-- Support for multiple Event Grid topics by sending events to `http://127.0.0.1:6500/<topic-name>/api/events`.
+- Support for multiple Event Grid topics by sending events to `http://127.0.0.1:6500/<topic-name>/api/events` (Push) or through the `EventGridClient` API (Pull).
```

Comment:
It's weird to have 2 different ways of presenting the same info (URL and client name). We can use a URL for both cases:

```suggestion
- Support for multiple Event Grid topics by sending events to `http://127.0.0.1:6500/<topic-name>/api/events` or `http://127.0.0.1:6500/topics/<topic-name>:publish`.
```